diff --git a/cmake/PyneMacros.cmake b/cmake/PyneMacros.cmake
index 59d86359..8f42425a 100644
--- a/cmake/PyneMacros.cmake
+++ b/cmake/PyneMacros.cmake
@@ -81,8 +81,10 @@ macro(pyne_setup_fortran)
     message("CMAKE_Fortran_COMPILER full path: " ${CMAKE_Fortran_COMPILER})
     message("Fortran compiler: " ${Fortran_COMPILER_NAME})
     message ("No optimized Fortran compiler flags are known, we just try -fpic...")
-    set(CMAKE_Fortran_FLAGS_RELEASE "-fpic")
-    set(CMAKE_Fortran_FLAGS_DEBUG   "-fpic")
+    if(NOT WIN32)
+      set(CMAKE_Fortran_FLAGS_RELEASE "-fpic")
+      set(CMAKE_Fortran_FLAGS_DEBUG   "-fpic")
+    endif()
   endif(Fortran_COMPILER_NAME MATCHES "gfortran.*")
 endmacro()
 
